common/src/main/scala/com/gu/media/aws/AwsCredentials.scala

```diff
@@ -44,10 +44,7 @@ object AwsCredentials {
   }
 
   private def assumeCrossAccountRole(instance: AWSCredentialsProvider, settings: Settings): AWSCredentialsProvider = {
-    val securityTokens = AWSSecurityTokenServiceClientBuilder
-      .standard()
-      .withCredentials(instance)
-      .build()
+    val securityTokens = new AWSSecurityTokenServiceClient(instance)
```

Comment:
Nope. The static credentials are only for DEV. CODE and PROD use the normal AWS injected ones